vault/resource_identity_entity_alias_test.go

```diff
@@ -0,0 +1,77 @@
+package vault
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/hashicorp/vault/api"
+)
+
+func TestAccIdentityEntityAlias(t *testing.T) {
```

Comment:
The tests won't build for me currently due to a dependency issue; I think master needs to be merged in.

Reply:
I thought I had merged master in, will take a look.